enterprise-suite/tests/smoke_prometheus

```diff
@@ -9,8 +9,7 @@ source smokecommon
 #
 
 set -e
-NAMESPACE="smoketest-prometheus"
-kubectl create namespace $NAMESPACE
+NAMESPACE=${NAMESPACE:=lightbend}
```

Comment:
I assume this change is fine.  I just figured there was _some_ reason for using the other namespace originally.  :-)  Didn't have the nerve to change it...